src/global.d.ts

```diff
@@ -3,4 +3,8 @@ declare interface Page {
   url: string;
 }
 
+declare interface Locale {
+  localeId: string;
```

Comment:
Is this enough for users to set the locale? Perhaps they also need the value of the locale, e.g. `nl`

Reply:
I've added the name and code for the language.